plenum/test/plugin/demo_plugin/__init__.py

```diff
@@ -0,0 +1,16 @@
+from plenum.common.messages.fields import FixedLengthField
+from plenum.test.plugin.demo_plugin.transactions import DemoTransactions
+from plenum.test.plugin.demo_plugin.constants import AUCTION_LEDGER_ID
+
+
+dummy_field_length = 10
+LEDGER_IDS = {AUCTION_LEDGER_ID, }
+CLIENT_REQUEST_FIELDS = {'fix_length_dummy':
+                         FixedLengthField(dummy_field_length,
+                         optional=True, nullable=True)}
+
+AcceptableWriteTypes = {DemoTransactions.AUCTION_START.value,
```

Comment:
Where is it used?

Reply:
`plenum/test/plugin/demo_plugin/test_plugin_request_handling.py`